src/content/hcp/docs-landing.json

```diff
@@ -1,5 +1,27 @@
 {
 	"pageSubtitle": "Learn and develop your knowledge of HashiCorp Cloud Platform with these tutorials and code resources.",
+	"iconCardGridItems": [
```

Comment:
Made this authorable...open to suggestions though if we'd prefer not to do this.